### ISSUE TITLE
Ubuntu 22.04 Beta无法安装Tencent Meeting，出现依赖问题

### ISSUE THREAD

**Issue by Timeral** [zq1997/deepin-wine#274]
![image](https://user-images.githubusercontent.com/22254456/162886530-18dd57b8-d5a7-4529-a979-75222bcadfcd.png)

尝试过按照CSDN的方法解决依赖问题但无果。

**Comment by YerongLi** [comment]
Met the same issue. Not sure how to fix this.

**Comment by ChengysTHU** [comment]
Tencent Meeting has deb packages. see https://meeting.tencent.com/download-center.html?from=1001

**Comment by YerongLi** [comment]
> Tencent Meeting has deb packages. see https://meeting.tencent.com/download-center.html?from=1001

Yeah, I am on debian 11/bookworm version I don't understand why when I run the linux version of the tencent meeting it does not run as normal.

    $ /opt/wemeet/wemeetapp.sh
    wemeet:WemeetRun
    wemeet:WemeetStop


	$ echo $XDG_CURRENT_DESKTOP
	lsb_release -a
	grep -rn '^\s*deb ' --include '*.list' /etc/apt/
	icewm
	No LSB modules are available.
	Distributor ID:	Debian
	Description:	Debian GNU/Linux bookworm/sid
	Release:	11-updates
	Codename:	n/a
	/etc/apt/sources.list.d/google-chrome.list:3:deb [arch=amd64] https://dl.google.com/linux/chrome/deb/ stable main
	/etc/apt/sources.list.d/slack.list:3:deb https://packagecloud.io/slacktechnologies/slack/debian/ jessie main
	/etc/apt/sources.list.d/debian.list:2:deb http://ftp.us.debian.org/debian/ bullseye main contrib non-free
	/etc/apt/sources.list.d/debian.list:3:deb http://security.debian.org/ bullseye-security main contrib non-free
	/etc/apt/sources.list.d/debian.list:22:deb http://deb.debian.org/debian/ testing main contrib non-free
	/etc/apt/sources.list.d/debian.list:23:deb http://deb.debian.org/debian/ testing-updates main contrib non-free
	/etc/apt/sources.list.d/debian.list:24:deb http://deb.debian.org/debian-security testing-security main
	/etc/apt/sources.list.d/debian-stable-updates.list:2:deb http://ftp.us.debian.org/debian/ bullseye-updates main contrib non-free
	/etc/apt/sources.list.d/vscode.list:3:deb [arch=amd64,arm64,armhf] http://packages.microsoft.com/repos/code stable main
	/etc/apt/sources.list.d/deepin-wine.i-m.dev.list:1:deb [trusted=yes] https://deepin-wine.i-m.dev /
	/etc/apt/sources.list.d/bullseye-backports.list:2:deb http://deb.debian.org/debian bullseye-backports main contrib non-free
	/etc/apt/sources.list.d/antix.list:2:deb http://mirrors.rit.edu/mxlinux/mx-packages/antix/bullseye bullseye main nosystemd nonfree

**Comment by Timeral** [comment]
> Tencent Meeting has deb packages. see https://meeting.tencent.com/download-center.html?from=1001

Already tried, but the pakage can't run on Ubuntu 22.04 beta, either.

`undefined symbol: wl_proxy_marshal_flags`

**Comment by zhuohoudeputao** [comment]
22.04上面微信有没有问题啊？

**Comment by Timeral** [comment]
> 22.04上面微信有没有问题啊？

可以正常收发消息，但有些表情显示不出来，点击表情闪退

**Comment by fenguoerbian** [comment]
> > Tencent Meeting has deb packages. see https://meeting.tencent.com/download-center.html?from=1001
> 
> Already tried, but the pakage can't run on Ubuntu 22.04 beta, either.
> 
> `undefined symbol: wl_proxy_marshal_flags`

In my machine it works when I use Xorg to login Gnome, but apparently slower than before on 20.04. If use Wayland when logging to Gnome, the Tecent meeting won't start.

**Comment by YaoLaiGang** [comment]
> 22.04上面微信有没有问题啊？

我直接装不上，依赖不满足

**Comment by ipangbo** [comment]
我也有相同的问题，22.04兼容性还需要适配

**Comment by shmily103** [comment]
ubuntu22.04正式版本微信使用正常。

**Comment by zhuohoudeputao** [comment]
昨天更新了，使用也挺正常

**Comment by fenguoerbian** [comment]
可以考虑使用Ukylin-wine的[腾讯会议](https://www.ubuntukylin.com/applications/109-cn.html)。Ukylin-wine采用的类似Appimage的打包方式，依赖问题没有那么多。我在Ubuntu22.04上可以正常安装。使用方面的话，一个是登录二维码/验证码图片刷不出来，只能密码登录，另一个就是会议聊天室里中文输入法切换不出来。

**Comment by FeyXieXzf** [comment]
腾讯会议的原生版本(deb版)本来就不支持wayland，在wayland启动的时候应该会提示不支持然后退出
